docs/software-development-kits/javascript/express.mdx

```diff
@@ -227,7 +227,7 @@ The Topaz / Aserto [authorizers](https://github.com/aserto-dev/topaz) exposes SS
 
 For a hosted authorizer that has a TLS certificate that is signed by a trusted Certificate Authority, this section isn't relevant because that TLS certificate will be successfully validated.
 
-In a development environment, the edge authorizer (either Topaz or the Aserto [sidecar](https://github.com/aserto-dev/sidecar)) automatically creates a set of self-signed certificates and certificates of the CA (certificate authority) that signed them. It places them in a well-known location on the filesystem, defaulting to `$HOME/.config/aserto/sidecar/certs/`. For Topaz this is `$HOME/.config/topaz/certs/`.
+In a development environment, the Edge Authorizer (either Topaz or the Aserto [sidecar](https://github.com/aserto-dev/sidecar)) automatically creates a set of self-signed certificates and certificates of the CA (certificate authority) that signed them. It places them in a well-known location on the filesystem, defaulting to `$HOME/.config/aserto/sidecar/certs/`. For Topaz this is `$HOME/.config/topaz/certs/`.
```

Comment:
this should remove the "sidecar" reference (the repo has been archived I believe).